### PR TITLE
fixed invalid error message

### PR DESCRIPTION
**Pull request by kapildave07**
## Description

fixed invalid error message

## Changes

- fixed invalid error message

## Attached Links

1. Jira ticket: https://investax.atlassian.net/browse/IXSPD1-1943
2. Documents:
3. UI Demo
image

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/pages/KYC/schema.ts [hunk 1]
```diff
@@ -327,26 +327,30 @@ export const corporateErrorsSchema = yup.object().shape({
...
+  }),
+taxNumber: yup
+  .string()
+  .nullable()
+  .when('taxIdAvailable', {
     is: true,
     then: yup.string().required('Required'),
-    otherwise: yup.string(),
+    otherwise: yup.string().nullable(),
   }),
```
Comment (thi-investax) [inline src/pages/KYC/schema.ts:352]: @kapildave07 is this correct indentation?

Comment (kapildave07) [reply src/pages/KYC/schema.ts:352]: @thi-investax yes it's correct

File: src/pages/KYC/schema.ts [hunk 2]
```diff
@@ -295,16 +295,16 @@ export const corporateErrorsSchema = yup.object().shape({
...
+  registrationNumber: yup.string().nullable().required('Required'),
   inFatfJurisdiction: yup.string().required('Required'),
 
   personnelName: yup.string().required('Required'),
-  designation: yup.string().required('Required'),
+  designation: yup.string().nullable().required('Required'),
   email: yup.string().email('Invalid email').required('Required'),
   phoneNumber: yup
-    .string()
+    .string().nullable()
```
Comment (thi-investax) [inline src/pages/KYC/schema.ts:307]: @kapildave07 why we make these fileds nullable but later on required?

Comment (kapildave07) [reply src/pages/KYC/schema.ts:307]: @thi-investax these fields are still required 
image

Avoid these errors we marked as a nullable 

image

**Comment by thi-investax** [comment]
@leonthuongto could you please help review this pr? Here kapil use both .nullable() and .required() but I think they conflict with each other.

**Pull request merged**